pillar.example

```diff
@@ -39,12 +39,14 @@ rabbitmq:
   config_files:
     # config_files, if you need some stuffs set directly in configuration
     # files, rather than via the light modules available.
-    rabbitmq.config:
-      # Need to add some template in rabbitmq/templates/rabbitmq.config
-      # templates not provided, as is probably too specific...
-      source: templates/relay-rabbitmq.config
+    rabbitmq.conf:
+      # source is relative to `rabbitmq` formula
+#      source: templates/rabbitmq.conf
```

Comment:
The `comments-indentation` error usually requires the indentation of the comment character to be in line with _the line below_, so in this case:

```suggestion
      # source: templates/rabbitmq.conf
```